git-redate

```diff
@@ -76,8 +76,10 @@ while [[ $# -ge 1 ]] ; do
             echo "  -h| --help      Show this help message."
             echo "  -v| --version   Show the version (${VERSION}) of git-redate"
             echo "  -l| --limit N   Process commits in chunks/batches of size N, default=20."
-            echo "Configuration (${SETTINGS_FILE}):"
-            cat "${SETTINGS_FILE}"
+            if [ -f "$SETTINGS_FILE" ]; then
```

Comment:
Yes, makes sense!
Please could you extend this to e.g.?
```
            echo "Configuration (${SETTINGS_FILE}):"
            if [ -f "$SETTINGS_FILE" ]; then
                cat "${SETTINGS_FILE}"
            else
                echo "does not exist yet."
            fi
            exit 0	
```
I'd like to let users know early that a config file will be created.